pyairtable/api/api.py

```diff
@@ -17,17 +18,24 @@ class Api(ApiAbstract):
         >>> api.all('base_id', 'table_name')
     """
 
-    def __init__(self, api_key: str, timeout=None):
+    def __init__(
+        self,
+        api_key: str,
+        timeout=None,
+        max_records_per_request=DEFAULT_MAX_RECORDS_PER_REQUEST,
+    ):
```

Comment:
This feels like an awkward pattern or code-smell, since it requires importing the default value and using it as a parameter default, which then must be passed and will override the default provided by the superclass.

A better pattern might be to just move the `*` explicit keyword-only argument separator to the `AbstractApi.__init__` from the `Table.__init__` and make both `timeout` and `max_records_per_request` into keyword-only arguments (this is my suggestion above) and then here (as well as `Base` & `Table`) just accept and pass a `**kwargs` dictionary. 

```suggestion
    def __init__(self, api_key: str, **kwargs: Any):
```
_NB, don't forget to add the import for `Any` from `typing` above_